tsconfig.json

```diff
@@ -42,12 +35,8 @@
     "jest",
     "razzle.config.js",
     "healthCheck.js",
-    "src/desktop/**/*.js"
+    "src/desktop/**/*.js",
+    "node_modules/react-use"
```

Comment:
Why do we need to add this here?
I suspect warning in console, coming from here...
<img width="480" alt="Screenshot 2023-01-14 at 07 30 23" src="https://user-images.githubusercontent.com/3015990/212445653-a7762410-9e7c-482e-8b61-efeb8a34808e.png">

Reply:
Not from here. Warning because we have too old typescript version

Reply:
I tried to fix it but it didn't help. Will return as it was